src/exam/dto/create-exam.dto.ts

```diff
@@ -0,0 +1,29 @@
+import { ApiHideProperty } from '@nestjs/swagger';
+import {
+  ArrayMinSize,
+  IsArray,
+  IsNumber,
+  IsOptional,
+  IsString,
+} from 'class-validator';
+
+export class CreateExamDto {
+  @IsString()
+  readonly title: string;
+
+  @IsString()
+  readonly description: string;
+
+  @IsNumber()
+  readonly price: number;
+
+  @ArrayMinSize(1)
+  readonly tags: string[];
+
+  @IsOptional()
+  @IsArray()
+  readonly samples: string[];
```

Comment:
I think we can remove this, right?

Reply:
I mean, sample questions should be added after the exam is created

Reply:
are sample questions subset of exam questions?